emperor/support_files/js/sceneplotview3d.js

```diff
@@ -263,6 +279,59 @@ define([
     });
   };
 
+  /**
+   * Builds a camera (for scatter or parallel plot)
+   */
+  ScenePlotView3D.prototype.buildCamera = function(viewType) {
+
+    var camera;
+    if (viewType === 'scatter')
+    {
+      // Set up the camera
+      var max = _.max(this.decViews.scatter.decomp.dimensionRanges.max);
+      var frontFrust = _.min([max * 0.001, 1]);
+      var backFrust = _.max([max * 100, 100]);
+
+      // these are placeholders that are
+      // later updated in updateCameraAspectRatio
+      camera = new THREE.OrthographicCamera(-50, 50, 50, -50);
+      camera.position.set(0, 0, max * 5);
+      camera.zoom = 0.7;
+    }
+    else if (viewType === 'parallel-plot')
+    {
+      var w = this.decModels.dimensionRanges.max.length;
+
+      // Set up the camera
+      camera = new THREE.OrthographicCamera(0, w, 1, 0);
+      camera.position.set(0, 0, 1); //Must set positive Z because near > 0
+      camera.zoom = 0.7;
+    }
+
+    return camera;
+  };
+
+  /**
+   * Builds a camera controller (for scatter or parallel plot)
+   */
+  ScenePlotView3D.prototype.buildCamController = function(viewType, cam, view) {
+    /**
+     * Object used to interact with the scene. By default it uses the mouse.
+     * @type {THREE.OrbitControls}
+     */
+    var control = new THREE.OrbitControls(cam,
```

Comment:
Can you join this and the next line? Doesn't look like these will exceed the 80 column limit.